src/data/nat/prime.lean

```diff
@@ -380,6 +380,16 @@ theorem prime.coprime_iff_not_dvd {p n : ℕ} (pp : prime p) : coprime p n ↔ 
 theorem prime.dvd_iff_not_coprime {p n : ℕ} (pp : prime p) : p ∣ n ↔ ¬ coprime p n :=
 iff_not_comm.2 pp.coprime_iff_not_dvd
 
+theorem prime.dvd_of_not_coprime {m n : ℕ} (h : ¬ coprime m n) :
```

Comment:
Could you make this an `iff` statement?

Reply:
You mean _not_ an iff statement like you have?

Reply:
No, I meant: could you prove `¬ coprime m n \iff ∃p, prime p ∧ p ∣ m ∧ p ∣ n`, and add that as a lemma (potentially replacing this lemma). `iff` statements are nice, because you can rewrite with them, or tell `simp` to apply them, and when doing so you know for certain you're not losing any information.